detector-data/detectors/HPS_PhysicsRun2019-v2-FEE-Pass0/compact.xml

```diff
@@ -0,0 +1,1086 @@
+          <unitvec name="Y" x="-1.3440e-04" y="1.0000e+00" z="-2.1271e-06" />
+          <unitvec name="Z" x="3.0350e-02" y="6.2051e-06" z="9.9954e-01" />
+        </SurveyVolume>
+        <SurveyVolume name="support_top_L13" desc="T13 ball basis in pivot frame:">
+          <origin x="0." y="-50.9272" z="-319.7481" />
+          <unitvec name="X" x="1." y="0." z="0." />
+          <unitvec name="Y" x="0." y="1." z="0." />
+          <unitvec name="Z" x="0." y="0." z="1." />
+        </SurveyVolume>
+        <SurveyVolume name="c_support_kin_L13t" desc="T13 pivot basis in box frame:">
+          <origin x="-36.9837" y="59.4777" z="694.2924" />
+          <unitvec name="X" x="9.9952e-01" y="4.2882e-05" z="-3.1132e-02" />
+          <unitvec name="Y" x="-5.2348e-05" y="1.0000e+00" z="-3.0325e-04" />
+          <unitvec name="Z" x="3.1132e-02" y="3.0473e-04" z="9.9952e-01" />
+        </SurveyVolume>
+        -->
+
+
+        <!-- Module support surface survey -->
+        <!-- not available for 2019 -->
+        <!-- I used the 2016 survey for L5, L6. L7 Top/Bottom have been refurbished. -->
+        <!-- Caution:: Ly5 in 2016 is Ly6 in 2019! -->
+
+
+        <SurveyVolume name="module_L5t" desc="Top L5 pin basis in U-channel fiducial frame:">
+          <origin x="-149.2748" y="53.2984" z="-9.568" />
+          <unitvec name="X" x="1.0000e+00" y="1.3154e-04" z="-1.6050e-04" />
+          <unitvec name="Y" x="1.3166e-04" y="-1.0000e+00" z="7.2035e-04" />
+          <unitvec name="Z" x="-1.6041e-04" y="-7.2037e-04" z="-1.0000e+00" />
+        </SurveyVolume>
+        <SurveyVolume name="module_L6t" desc="Top L6 pin basis in U-channel fiducial frame:">
+          <origin x="-149.2454" y="56.3526" z="190.4708" />
+          <unitvec name="X" x="1.0000e+00" y="2.0492e-04" z="-2.5400e-04" />
+          <unitvec name="Y" x="2.0495e-04" y="-1.0000e+00" z="1.1388e-04" />
+          <unitvec name="Z" x="-2.5398e-04" y="-1.1393e-04" z="-1.0000e+00" />
+        </SurveyVolume>
+        <SurveyVolume name="module_L5b" desc="Bottom L5 pin basis in U-channel fiducial frame:">
+          <origin x="-149.1868" y="-53.3685" z="9.5839" />
+          <unitvec name="X" x="1.0000e+00" y="1.1536e-04" z="-1.7794e-04" />
+          <unitvec name="Y" x="-1.1535e-04" y="1.0000e+00" z="9.3250e-05" />
+          <unitvec name="Z" x="1.7795e-04" y="-9.3229e-05" z="1.0000e+00" />
+        </SurveyVolume>
+        <SurveyVolume name="module_L6b" desc="Bottom L6 pin basis in U-channel fiducial frame:">
+          <origin x="-149.1631" y="-56.3514" z="209.5891" />
+          <unitvec name="X" x="1.0000e+00" y="6.1076e-05" z="-8.7472e-05" />
+          <unitvec name="Y" x="-6.1071e-05" y="1.0000e+00" z="5.2874e-05" />
+          <unitvec name="Z" x="8.7475e-05" y="-5.2869e-05" z="1.0000e+00" />
+        </SurveyVolume>
+
+
+        <!-- Sensor position survey  -->
+
+        <!-- For 2019 only the front sensors L1, L2, L3 and L4 have been surveyed -->
```

Comment:
Do we need to keep the old survey positions for any reason? Aren't they in an older compact? Maybe it's best to just delete this too.

Reply:
Hey Omar, 
I need the old positions for the back of the detector.